### ISSUE TITLE
ubuntu 22.04

### ISSUE THREAD

**Issue by adammaj1** [eoyilmaz/displaycal-py3#284]
HI

I have installed app on Ubuntu 22.04, but i can run it only inside python : 

```
displaycal
displaycal: command not found
a@zalman:~$ python3 -m venv ./displaycal_venv
a@zalman:~$ displaycal
displaycal: command not found
a@zalman:~$ source ./displaycal_venv/bin/activate
(displaycal_venv) a@zalman:~$ displaycal
Acquired lock file: <DisplayCAL.main.AppLock object at 0x7f8d23322a40>
displaycal 3.9.11 2023-07-18T20:43:25Z
ubuntu 22.04 jammy x86_64
Python 3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0]
Faulthandler 
wxPython 4.2.1 gtk3 (phoenix) wxWidgets 3.2.2.1
Encoding: utf-8
File system encoding: utf-8
Loading /home/a/.config/DisplayCAL/DisplayCAL.ini
Existing client using port 15411
Connecting to 15411...
Connection to 127.0.0.1:15411 failed: [Errno 111] Connection refused
listening
writing to lock file: port: 35629
Starting up...
Warning: Missing bitmap 'theme/icons/256x256/DisplayCAL'
Warning: Missing bitmap 'theme/icons/48x48/DisplayCAL'
Warning: Missing bitmap 'theme/icons/32x32/DisplayCAL'
Warning: Missing bitmap 'theme/icons/16x16/DisplayCAL'
SDL2: libSDL2-2.0.so.0
Audio module: wx 4.2.1
Enumerating display devices and communication ports...
/usr/bin
ArgyllCMS 2.2.0
Argyll has virtual display support
...ok.
````

Is it OK ?

**Comment by domi84** [comment]
Hello, I just wanted to point my troubleshooting with Ubuntu (Gnome) 22.04, might be useful for others.

I managed to install it using this procedure https://discuss.pixls.us/t/installing-displaycal-via-pip-on-ubuntu-22-04-based-distros-kde-neon/35908/5, so:

```
sudo apt-get install python3-pip python3-venv libgtk-3-dev libxxf86vm-dev libglib2.0-dev wxpython-tools git
pip install DisplayCAL==3.9.8
```
(I couldn't install the last version available on pip)

I had also to install an updated version of wxpython:
`pip install wxpython==4.1.1`

I was getting this issue (not able to install Agryll):
https://github.com/eoyilmaz/displaycal-py3/issues/291
So as suggested I downloaded the binary for linux https://www.argyllcms.com/downloadlinux.html and extracted them

Finally I got this issue:
https://github.com/eoyilmaz/displaycal-py3/issues/113
It writes the profile, but can't apply it

```
DisplayCAL: Trying device ID 'xrandr-HP Inc.-HP V24i-1CR11004XM'
Writing /home/domenico/.local/share/icc/HP V24i #1 2023-09-17 20-44 2.2 F-S XYZLUT+MTX.icc
CDError("gi.repository.GLib.Error: GLib.Error('GDBus.Error:org.freedesktop.ColorManager.NotFound: profile id 'icc-7502cfef8f24a084612a018ec1cdfe99' does not exist', 'g-io-error-quark', 36)")
```
I bypassed it opening the Gnome Setting -> Color, importing the icc file created (/home/USER/.local/share/icc/).

**Comment by eoyilmaz** [comment]
Closing this. Please create new issue tickets or open this one if you still have the same problem with the soon to be released DisplayCAL 3.9.12

**Comment by eoyilmaz** [comment]
3.9.12 is released, fyi.
